tests/nightly/test_large_vector.py

```diff
@@ -56,15 +55,12 @@ def test_ndarray_random_uniform():
 
 @with_seed()
 def test_ndarray_random_randint():
-    a = nd.random.randint(100, 10000, shape=LARGE_X)
-    assert a.shape == (LARGE_X,)
     # check if randint can generate value greater than 2**32 (large)
-    low_large_value = 2**32
-    high_large_value = 2**34
-    a = nd.random.randint(low_large_value, high_large_value, dtype=np.int64)
-    low = mx.nd.array([low_large_value], dtype='int64')
-    high = mx.nd.array([high_large_value], dtype='int64')
-    assert a >= low and a < high
+    low = 4294967296
```

Comment:
`2**32 == 4294967296`
It will be more specific to use `2**32`.

Reply:
Done!